src/regression_tests/utils.ts

```diff
@@ -0,0 +1,183 @@
+import { SpvWalletClient } from '../client';
+import { generateKeys } from '../utils/keys';
+import { TransactionFilter } from '../filters';
+import { QueryParams, Recipients, Tx, Txs } from '../types';
+
+const CLIENT_ONE_URL_ENV_VAR = 'CLIENT_ONE_URL';
+const CLIENT_TWO_URL_ENV_VAR = 'CLIENT_TWO_URL';
+const CLIENT_ONE_LEADER_XPRIV_ENV_VAR = 'CLIENT_ONE_LEADER_XPRIV';
+const CLIENT_TWO_LEADER_XPRIV_ENV_VAR = 'CLIENT_TWO_LEADER_XPRIV';
+const AT_SIGN = '@';
```

Comment:
I'm not convinced that this `AT_SIGN` is usefull and makes the code more readable.
I would suggest to just use the `@`, like that:
```typescript
const url = paymailAlias + '@' + domain
```

Reply:
I see your point. It was supposed to look the most similar to go regression tests. Do you think it should be changed there as well?